snap/snapcraft.yaml

```diff
@@ -0,0 +1,148 @@
+name: opensearch-dashboards # you probably want to 'snapcraft register <name>'
```

Comment:
should all template comments  be removed: `# you probably want to 'snapcraft register <name>'` ?